sdf/1.9/bounding_box_camera.sdf

```diff
@@ -0,0 +1,17 @@
+<element name="boundingbox" required="0">
+  <description>These elements are specific to an boundingbox camera sensor.</description>
+
+  <element name="box_type" type="string" default="2d" required="0">
+    <description>
+      The boundingbox type of the boundingbox camera, the valid options:
+        - 2d | visible_2d | visible_box_2d: a visible 2d box mode which provides and axis aligned 2d boxes
+                                            on the visible parts of the objects
+
+        - full_2d | full_box_2d: a full 2d box mode which provides an axis aligned 2d boxes that fills the
+                                 object dimentions even if it had an occluded part
+
+        - 3d: a 3d mode which provides oriented 3d boxes
```

Comment:
Full, because they are a real 3d position and dimentions ... there is no meaning in visible concept here ... no body will suppose the visible concept in the 3d